recipes/pytest_parallel_mpi/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytest_parallel_mpi" %}
+{% set version = "1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/onera/pytest_parallel/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 666ecb0d4fbdfe07b25dca538e0dd105caf7cb49c52f51cf459531b4a37801f2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - pytest ==6.2.5
```

Comment:
Where is this strict requirement come from? It doesn't look like upstream has it.

Reply:
Upstream has this strict requirement for the release but not the current master branch (development not released.)